src/app-gocardless/banks/spk_worms_alzey_ried_malade51wor.js

```diff
@@ -0,0 +1,63 @@
+import Fallback from './integration-bank.js';
+
+import { printIban, amountToInteger } from '../utils.js';
+import { formatPayeeName } from '../../util/payee-name.js';
+
+/** @type {import('./bank.interface.js').IBank} */
+export default {
+  ...Fallback,
+
+  institutionIds: ['SPK_WORMS_ALZEY_RIED_MALADE51WOR'],
+
+  accessValidForDays: 90,
+
+  normalizeAccount(account) {
+    return {
+      account_id: account.id,
+      institution: account.institution,
+      mask: account.iban.slice(-4),
+      iban: account.iban,
+      name: [account.name, printIban(account)].join(' '),
+      official_name: account.product,
+      type: 'checking',
+    };
+  },
+
+  normalizeTransaction(transaction, _booked) {
+    const date = transaction.bookingDate || transaction.valueDate;
+    if (!date) {
+      return null;
+    }
+
+    let remittanceInformationUnstructured;
+
+    if (transaction.remittanceInformationUnstructured) {
+      remittanceInformationUnstructured =
+        transaction.remittanceInformationUnstructured;
+    } else if (transaction.remittanceInformationStructured) {
+      remittanceInformationUnstructured =
+        transaction.remittanceInformationUnstructured;
+    } else if (transaction.remittanceInformationStructuredArray?.length > 0) {
+      remittanceInformationUnstructured =
+        transaction.remittanceInformationStructuredArray?.join(' ');
+    }
+
+    transaction.remittanceInformationUnstructured =
+      remittanceInformationUnstructured;
```

Comment:
This could be made a little more concise and readable. I also think there might be a typo in the first else if, should it say `remittanceInformationUnstructured = transaction.remittanceInformationStructured;`?

```suggestion
    transaction.remittanceInformationUnstructured =
      transaction.remittanceInformationUnstructured ??
      transaction.remittanceInformationStructured ??
      transaction.remittanceInformationStructuredArray?.join(' ');
```